sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/BlobDownloadHeaders.java

```diff
@@ -12,27 +12,107 @@
 import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement;
 import java.time.OffsetDateTime;
 import java.util.Map;
+import java.util.HashMap;
 
 /**
  * Defines headers for Download operation.
  */
 @JacksonXmlRootElement(localName = "Blob-Download-Headers")
 @Fluent
 public final class BlobDownloadHeaders {
+
+    public BlobDownloadHeaders() {
+
+    }
+
+    public BlobDownloadHeaders(com.azure.storage.blob.implementation.models.BlobDownloadHeaders headers) {
```

Comment:
Can we add a comment somewhere just for future reference for why we have these two classes?